inc/class-saml.php

```diff
@@ -433,7 +437,9 @@ private function getSettingsWithoutCertificatesAndPrivateKey() {
 	 */
 	public function getUsernameByAttributes( $attributes ) {
 		if ( isset( $attributes[ self::SAML_MAP_FIELDS['uid'] ] ) ) {
-			return $attributes[ self::SAML_MAP_FIELDS['uid'] ][0];
+			$uid = $attributes[ self::SAML_MAP_FIELDS['uid'] ][0];
```

Comment:
UID field could be for example: `johndoe@fakeuniversity.pub`

Reply:
I see. It shouldn't be this value (we're requesting just the prefix part), but it looks like some users are sending us this, I suppose.